CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -14,10 +14,13 @@ Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/
 
 - [Auto sync members in v3store for the issues which have already been affected by #19557](https://github.com/etcd-io/etcd/pull/19636).
 - [Move `client/internal/v2` into `server/internel/clientv2`](https://github.com/etcd-io/etcd/pull/19673).
+- [Replace ExperimentalMaxLearners with a Feature Gate](https://github.com/etcd-io/etcd/pull/19560).
+- Fix [infinite loop that may cause high CPU usage in grpcproxy](https://github.com/etcd-io/etcd/pull/19561).
```

Comment:
checked existing release and I think we should create new section `etcd grpc-proxy` for this fix

Reply:
Ah, right. Thanks, good catch.